### DISCUSSION TITLE
Feature: Support encrypt

### DISCUSSION THREAD

**Discussion by aquarius-wing** [electric-sql/electric#556] [Feature requests]
As said in the title, could you create a feature that encrypts data before syncing it to the remote server? 

I believe this is a natural thing for a locally prioritized application.

**Comment by alco** [comment]
Hey @aquarius-wing. Thanks for bringing it up!

We have this feature on our internal roadmap. No comment on the timeline for now.

**Comment by aquarius-wing** [comment]
> Hey @aquarius-wing. Thanks for bringing it up!
> 
> We have this feature on our internal roadmap. No comment on the timeline for now.

Nice to hear that!

**Comment by balegas** [comment]
Turning this into a feature request discussion

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
